bkg/v2/BKG_v2.0.0.yaml

```diff
@@ -6697,16 +6697,14 @@ components:
           description: |
             Number of requested equipment units.
           example: 3
-        emptyContainerPositioningDateTime:
-          type: string
-          format: date-time
+        emptyContainerPositions:
+          type: array
           description: |
-            The date and time requested by the shipper for the positioning of the empty container(s) at the `Empty Container Positioning Location` (ECP), if provided, or the `Place of Receipt` (PRE) if ECP location is not provided.
+            Possibility to add multiple `ECP` (Empty Container Position) locations. If multiple `ECP` locations are provided, all locations must be within the same city.
```

Comment:
Is it Empty Container Position or Positioning?

Reply:
correct - it is `possitioning`